### PR TITLE
Gerrit-catalog-plugin: Fix incorrect main entry

### PR DESCRIPTION
**Pull request by anicke**
Signed-off-by: Niklas Aronsson <niklasar@axis.com>

## Hey, I just made a Pull Request!

Plugin can not be loaded due to incorrect "main" path in the "publishConfig".

#### :heavy_check_mark: Checklist

- [X] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [X] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: .changeset/loud-peaches-warn.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-gerrit': minor
```
Comment (benjdlambert) [inline .changeset/loud-peaches-warn.md:2]: Think this should be a `patch` change.

**Review by benjdlambert — COMMENTED** [review]
Thanks for this! Just one nit on the changeset.

File: .changeset/loud-peaches-warn.md (hunk 1 above)
Comment (anicke) [reply .changeset/loud-peaches-warn.md]: Agreed!

**Review by benjdlambert — APPROVED** [review]

**Comment by benjdlambert** [comment]
Thanks! And welcome to the community! 👍 :pray:

**Pull request merged**